source/Audio.cpp

```diff
@@ -110,6 +117,17 @@ namespace {
 	shared_ptr<Music> previousTrack;
 	int musicFade = 0;
 	vector<int16_t> fadeBuffer;
+	double musicVolume = 1.;
+	double playlistVolumeModifier = 0.;
+
+	const Playlist *currentPlaylist = nullptr;
+	const Track *currentPlaylistTrack = nullptr;
+	Track::GameState oldState = Track::GameState::IDLE;
+
+	std::chrono::time_point<std::chrono::high_resolution_clock> onEnd;
```

Comment:
```suggestion
	chrono::time_point<chrono::high_resolution_clock> onEnd;
```
`std::`s are unnecessary